### ISSUE TITLE
Pulling the speed knob on A330-GE doesn’t change speed to automatic mode. 

### ISSUE THREAD

**Issue by CaptainCrazyGaming0** [JMaverick16/Airbus_A330#15]
I first tried flying the A330-300GE and when I tried to pull the speed knob instead of changing the speed to automatic mode, it just changed to my current speed. I’ve only tried flying it once so I don’t know if it was a one time thing or a bug.

**Comment by legoboyvdlp** [comment]
Did you try pushing?

**Comment by CaptainCrazyGaming0** [comment]
Yes, I left clicked but that just increased the speed. Then I tried middle-clicking but it the speed set to the current speed.

**Comment by CaptainCrazyGaming0** [comment]
Right-clicking did nothing

**Comment by FirstOfficerDelta** [comment]
When did you do it?
Also, did you try setting your speed in the A/P dialog?

**Comment by CaptainCrazyGaming0** [comment]
I did not try setting my speed using the A/P menu but used the Flight Control Unit aka the autopilot control panel in the 3D cockpit.

**Comment by FirstOfficerDelta** [comment]
I mean when during the flight? Is there anything you did to the throttle?
Also, try using the dialog, however I am not sure by what automatic mode are you talking about: auto throttle does not activate? Or you cannot set your auto throttle to fly the speed you indicated in the FCU? Or you cannot set your auto throttle such that the dialog for the A/T in the FCU shows  “- - -“?

**Comment by CaptainCrazyGaming0** [comment]
1. Just after takeoff. 
2. After TOGA, I put it to the climb setting. 
3. Ok I will try using the dialog. 
4. Autothrottle does activate is just that normally when you push the knob (not pull, my bad) you get a sign like this "---·" which means that autopilot will control the speed based on flap settings and FMC cruise speed and the pilots don't choose the speed. 
5. When I try pushing (left-clicking) the speed just increases, unlike the heading knob where pushing makes the autopilot fly based on FMC waypoints.

**Comment by CaptainCrazyGaming0** [comment]
![IMG_0566](https://user-images.githubusercontent.com/51228980/97952653-30df9580-1dd9-11eb-8069-065fccf8e9ef.jpeg)

**Comment by CaptainCrazyGaming0** [comment]
Got that from the online "A330 Flight Deck and Systems Briefing for Pilots" Here is the link: [http://www.smartcockpit.com/docs/A330_Flight_Deck_and_Systems_Briefing_For_Pilots.pdf](url)

**Comment by CaptainCrazyGaming0** [comment]
I believe this is what happens on the A33X by Octal450

**Comment by FirstOfficerDelta** [comment]
Ah, that makes more sense, I have not flown the A330 in a while, but did you set up the MCDU?

**Comment by CaptainCrazyGaming0** [comment]
Yes, I set it up as much as I could, some buttons don’t work and just end up with a “NOT ALLOWED” sign at the bottom of the MCDU. I also set up the route planner and activated it.

**Comment by FirstOfficerDelta** [comment]
Not sure, but did you find a guide in Airbus_A330/Resources/A33X-Notes.pdf?

**Comment by CaptainCrazyGaming0** [comment]
> Not sure, but did you find a guide in Airbus_A330/Resources/A33X-Notes.pdf?

Ok, I will find it.

**Comment by FirstOfficerDelta** [comment]
Within that guide, upon final approach, it says to set the aircraft’s auto throttle into managed mode.

Additionally, the guide should show you which parts of the MCDU work in JMav’s A330, but do take note that for this case, JMav’s A330 is based on the IDG-A33X, so use the IDG tags, and not the TOH tags.

**Comment by CaptainCrazyGaming0** [comment]
Ok, thank u so much
